.werf/werf-dev-install.yaml

```diff
@@ -103,3 +103,5 @@ shell:
 
     echo 'eval "$(dhctl --completion-script-bash)"' >> /etc/bashrc
     echo '{{- env "CI_COMMIT_TAG" | default "dev" }}' > /deckhouse/version
+
+    apt-get update && apt-get install -y locale && apt-get clean
```

Comment:
move this commands in `beforeInstall` stage pls

Reply:
Done